include/snabbkaffe.hrl

```diff
@@ -212,7 +210,11 @@
 
 -else. %% SNK_COLLECTOR
 
--define(tp(LEVEL, KIND, EVT), ?LOG(LEVEL, EVT #{?snk_kind => KIND})).
+-define(tp(LEVEL, KIND, EVT),
+        logger:log(LEVEL, EVT#{ ?snk_kind => KIND
+                              , mfa => {?MODULE, ?FUNCTION_NAME, ?FUNCTION_ARITY}
+                              , line => ?LINE
```

Comment:
It would be nice to include process metadata as well.

Reply:
what's process metadata ?

Reply:
Logger stores some additional data in the process dictionary (see `http://erlang.org/doc/man/logger.html#set_process_metadata-1`). Logger's `?LOG` macro attaches the contents of the process metadata to all the logs emitted by the process, that could be used e.g. to filter log messages by domain and what not.

Reply:
I believe it's added for `logger:log` calls too.
https://github.com/erlang/otp/blob/OTP-23.2.6/lib/kernel/src/logger.erl#L1070-L1084
both `?LOG` macro and `logger:log` calls the same `log_allowed`

Reply:
You're right, I misremembered how the LOG macro works.